studio/.nvmrc

```diff
@@ -1 +1 @@
-16
+18
```

Comment:
Would you mind also updating the root `.nvmrc` to match this one?

Reply:
Node engine in package.json is

``` "node": ">=18.0.0 <19.0.0"```

So probably best to have a lenient "18" in nvmrc?

Reply:
Yeah I reckon!